pertpy/tools/_mixscape.py

```diff
@@ -41,6 +41,7 @@ def perturbation_signature(
         adata: AnnData,
         pert_key: str,
         control: str,
+        ref_selection_mode: Literal["nn", "manual"] = "nn",
```

Comment:
My intuition tells me that these parameter names are not optimal. Instead of `manual` should it be `control` or `inferred` or `split_by`?